payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentResult.kt

```diff
@@ -0,0 +1,36 @@
+package com.stripe.android.payments.paymentlauncher
+
+import android.content.Intent
+import android.os.Parcelable
+import androidx.core.os.bundleOf
+import com.stripe.android.StripeIntentResult
+import com.stripe.android.model.StripeIntent
+import kotlinx.parcelize.Parcelize
+
+/**
+ * Result to be passed to the callback of [PaymentLauncher]
+ */
+sealed class PaymentResult : Parcelable {
+    @Parcelize
+    class Completed(
+        val intentResult: StripeIntentResult<StripeIntent>
+    ) : PaymentResult()
```

Comment:
This should be an `object` without any properties (i.e. not include the underlying `PaymentIntent`), just like in `PaymentSheet`